src/ziggurat/mapper.clj

```diff
@@ -80,7 +80,7 @@
             (sentry/report-error sentry-reporter e (str "Channel execution failed for " topic-entity-name " and for channel " channel-name))
             (metrics/increment-count default-namespace failure-metric additional-tags)))))))
 
-(defrecord MessagePayload [message topic-entity])
+(defrecord MessagePayload [message topic-entity headers])
 
 (s/defschema message-payload-schema
   {:message                      s/Any
```

Comment:
you'll also need to change the defschema as you changed the MessagePayload

Reply:
is it mandatory? because we dont set it in consumers